### PR TITLE
Allow access with apikey to some GQL queries

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Summary

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: lib/sanbase/auth/apikey/apikey.ex [hunk 1]
```diff
@@ -32,7 +32,12 @@ defmodule Sanbase.Auth.Apikey do
          true <- Hmac.apikey_valid?(token, apikey) do
       UserApikeyToken.user_by_token(token)
     else
-      _error ->
+      false ->
+        Logger.info("Apikey #{apikey} is not valid")
+        {:error, "Apikey not valid or malformed"}
+
+      error ->
+        Logger.info("Apikey #{apikey} cannot be splitted - #{inspect(error)}")
```
Comment (tspenov) [inline lib/sanbase/auth/apikey/apikey.ex]: I think `splitted` should be `split` :)

File: lib/sanbase_web/graphql/schema.ex [hunk 2]
```diff
@@ -389,7 +391,10 @@ defmodule SanbaseWeb.Graphql.Schema do
       arg(:interval, :string, default_value: "1d")
       arg(:result_size_tail, :integer, default_value: 0)
 
-      middleware(JWTAuth, san_tokens: 1000)
+      middleware(MultipleAuth, [
+        {JWTAuth, san_tokens: 1000},
+        {ApikeyAuth, san_tokens: 1000}
```
Comment (tspenov) [inline lib/sanbase_web/graphql/schema.ex]: Isn't the `MultipleAuth ` working only with `san_tokens` this way ?

Comment (tspenov) [reply lib/sanbase_web/graphql/schema.ex]: Oh thats only for the emojis

Comment (IvanIvanoff) [reply lib/sanbase_web/graphql/schema.ex]: The `MultipleAuth` accepts either `middleware` or `{middleware, config}`

Comment (tspenov) [reply lib/sanbase_web/graphql/schema.ex]: Yep, was looking out of context

**Pull request merged**